charts/fission-core/values.yaml

```diff
@@ -197,3 +197,9 @@ canaryDeployment:
 # Use these flags to enable opentracing, the variable is endpoint of Jaeger collector in the format shown below
 #traceCollectorEndpoint: "http://jaeger-collector.jaeger.svc:14268/api/traces?format=jaeger.thrift"
 #traceSamplingRate: 0.75
+
+## Message Queue Trigger Kind, KEDA: enable and configuration
+mqt_keda:
+  enabled: false
+  connector_images:
+    kafka: fission/keda-kafka:1
```

Comment:
Same comment as the fission-all/values.yaml

Reply:
done